libs/framework/src/celix_libloader.c

```diff
@@ -34,13 +33,20 @@ celix_library_handle_t* celix_libloader_open(celix_bundle_context_t *ctx, const
     if (noDelete) {
         flags = RTLD_LAZY|RTLD_LOCAL|RTLD_NODELETE;
```

Comment:
There is a related issue #492, which may be fixed in another PR.

Reply:
To be honest, I do not remember exactly. So good to look into this and improves this.

I think this had to with circular symbol usage between libraries, but that should not be present or IMO allowed.